Upgrade to jsonld ^5.0.2. (#102)
digitalbazaar/bedrock-ledger-node: CHANGELOG.md

# bedrock-ledger-node ChangeLog

> ## 15.2.0 - TBD

Reviewer: ```suggestion
## 16.0.0 - TBD
```

### Changed

> - Upgraded `jsonld` to `^5.2.0`.

Author: @davidlehn @dlongley are we sure this is a major change? @davidlehn seems to be suggesting the canonize outputs are similar between versions and the differences I saw is mostly do to different values in the created timestamp.

> - Upgraded `jsonld` to `^5.2.0`.

Reviewer: @dlongley @gannan08 @mattcollier: It appears this could be just a minor update based on the tests (including those added in https://github.com/digitalbazaar/bedrock-ledger-node/pull/105 / https://github.com/digitalbazaar/bedrock-ledger-node/pull/106). Are there other considerations that should make this a major release?  For instance, just to avoid potential unknown issues until a manual version update occurs.

> - Upgraded `jsonld` to `^5.2.0`.

Reviewer: @aljones15 In a call it was decided to use a major version. Basically just adds a bit of safety since we don't know how to test for potential unknown issues.

> - Upgraded `jsonld` to `^5.2.0`.

Reviewer: ```suggestion
- **BREAKING**: Upgraded `jsonld` to `^5.2.0`. Marked as a breaking major change to help avoid any unknown upgrading issues in live systems due to potential JSON-LD edge case processing changes. No issues are expected in normal use cases.
```

# bedrock-ledger-node ChangeLog

## 16.0.0 - TBD

### Changed

> - **BREAKING**: Upgraded `jsonld` to `^5.2.0` this might cause the `canonizedBytes`
>   & `hash` of records to change.

Reviewer: Could use a reword as is, but also I think readers won't know what to do with that statement.  Perhaps a hybrid with the text I had proposed could help.  I tried to not delve into the details and suggest we don't think there are issues.
```suggestion
- **BREAKING**: Upgraded `jsonld` to `^5.2.0`. Marked as a breaking major change to help avoid any unknown upgrading issues in live systems due to potential JSON-LD edge case processing changes. No issues are expected in normal use cases. If such an issue exists, it might cause the `canonizedBytes` and `hash` of records to change.
```
